recipes/metilene/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+make -j ${CPU_COUNT}
+cp metilene ${PREFIX}/bin
+cp metilene_input.pl ${PREFIX}/bin
+cp metilene_output.pl ${PREFIX}/bin
+cp metilene_output.R ${PREFIX}/bin
+install manual.pdf ${PREFIX}/share/doc/metilene
```

Comment:
We do not create the directory yet. One could e.g. add
install -d ${PREFIX}/share/doc/metilene
to explicitely create the dir or run
install -D manual.pdf ${PREFIX}/share/doc/metilene/manual.pdf